app/models/comment.rb

```diff
@@ -0,0 +1,5 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User',
+                      foreign_key: 'author_id'
+  belongs_to :post, counter_cache: :comments_count
+end
```

Comment:
- [ ] Comment model should have a method that updates the comments counter for a post.
```ruby
def update_comments_counter
  post.increment!(:comments_counter)
end
```